pkg/util/mount/nsenter_mount.go

```diff
@@ -175,8 +175,10 @@ func (n *NsenterMounter) IsLikelyNotMountPoint(file string) (bool, error) {
 		glog.V(5).Infof("findmnt: directory %s does not exist", file)
 		return true, err
 	}
-
-	args := []string{"--mount=/rootfs/proc/1/ns/mnt", "--", n.absHostPath("findmnt"), "-o", "target", "--noheadings", "--target", file}
+	// Add --first-only option in case multiple mount points exist.
```

Comment:
i'm a little concerned about `--first-only`.  Can you make an argument as to why we should ignore multiple mount points?